interfaces/automated-testing/scd/scd.yaml

```diff
@@ -35,6 +35,402 @@ components:
           type: string
           enum: [Starting, Ready]
           example: Ready
+    ################################################################################
+    #################### Start of ASTM-standard definitions    #####################
+    #################### interfaces/astm-utm/Protocol/utm.yaml #####################
+    ################################################################################
+    UUIDv4Format:
+      description: >-
+        String whose format matches a version-4 UUID according to RFC 4122.
+      maxLength: 36
+      minLength: 36
+      type: string
+      format: uuid
+      pattern: >-
+        ^[0-9a-fA-F]{8}\\-[0-9a-fA-F]{4}\\-4[0-9a-fA-F]{3}\\-[8-b][0-9a-fA-F]{3}\\-[0-9a-fA-F]{12}$
+      example: 03e5572a-f733-49af-bc14-8a18bd53ee39
+    EntityID:
```

Comment:
This interface doesn't use the EntityID schema. Either it should be removed or the parameter [flight_id](https://github.com/interuss/dss/pull/644/files#diff-7bc4ea3fc66c8059be4c6525f467ac28a80e6cbedc5af583c4548a205b80ef5dR485) could be updated to make use of it.